content/articles/javascript-math-anim/index.md

```diff
@@ -0,0 +1,197 @@
+
+We give the `container` class a 100% width, a height of 100 view height, and a flex display. We also justify the content and align everything to the center. This centers our content in the `container` div horizontally and vertically.
+
+The `circleimage` is given a `z-index` of 2 to overlay it over everything just in case the `sphere` overlaps with the image due to responsiveness. The margins are set to make it fit at almost the center. Again, the border-radius is set to 50% to give it a circular look of a rotating object.
+
+Our 'sphere' is not perfectly at the center of the rotation if you are keen. You can fine-tune it using your styling.
+
+### The full code
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+        .circle {
+          display: block;
+          background: black;
+          border-radius: 100%;
+          height: 100px;
+          width: 100px;
+
+          background: radial-gradient(circle at center,#31f541, #000);
+        }
+        .container {
+          width: 100%;
+          height: 100vh;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+        }
+        .circleimage{
+            z-index: 2; 
+            position: absolute; 
+            margin-left: 50%; 
+            margin-top: 50vh; 
+            border-radius: 50%;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <img class="circleimage" src="https://sacco.terrence-aluda.com/sacco/images/blue.png"> 
+        <figure class="circle"></figure>
+    </div>
+</body>
+<script>
+//the image variable
+       let image = document.querySelector("img");
+       //90 degrees
+       let angle = Math.PI / 2;
+       //timestamp variables
+       let prevTime = null;
+       let currentTime = Date.now();
+
+       function animate(currentTimeParam, prevTimeArg) {
+
+           if (lastTime != null) {
+              angle += (currentTimeParam - prevTimeArg) * 0.004;
+              lastTime = currentTimeParam ;
+           }
+           image.style.top = (Math.sin(angle) * 150) + "px";
+           image.style.left = (Math.cos(angle) * 150) + "px";
+           requestAnimationFrame(currentTime => animate(currentTime, prevTime));
+       }
+       requestAnimationFrame(animate);
+    </script>
+</html>
+```
+### Applications of this
+
+The most suitable areas for using these, that is after modifying it, include a circular progress indicator, a rotor in an aircraft/drone in a game, etc. These ideas are just some of many things you can do if you are creative and manipulate this idea well enough.
+
+### Conclusion
+
+We first looked at how the sin and cos functions are used in circles. Next, we created a simple script in JavaScript to show the creation of an animation. I hope you get the chance to creatively build onto this simple program in a fun way. Have a great read.
```

Comment:
I recommend that you flesh out the conclusion a little more. A good conclusion for a technical tutorial should, in my opinion:

1. Give a very general description of everything you did in the guide
2. Give the general steps you went through in the guide in order
3. Suggest ways the reader can learn more or practice
4. Give a nice *goodbye*

> Note that I ordered these in chronological order of what you should do in a conclusion

You did most of this just fine it's mostly the first point I listed there that I would probably include. Also, for the *goodbye* in the end, I don't think *have a great read* makes sense here because they already finished reading the article, it sounds as though you are saying they are about to read it again or something. Something like *happy coding!* should do just fine.

Lastly, if you can it would be nice if you can suggest more ways for the reader to practice or learn more about this stuff (maybe ask them to build something in particular) but that's optional.